### DISCUSSION TITLE
Add random comments between multi-lines tags ?

### DISCUSSION THREAD

**Discussion by Kaldor37** [apidoc/apidoc#1271] [Q&A]
Is there any way to add random comments between multi-lines apidoc tags ? Is there some kind of multi-line termination or escaping ?

Here is a silly example about what I'm trying to do : 
```
/**
 * @api {post} /test Something to post
 * @apiName Something
 * @apiDescription Some description
 * @apiGroup TheGroup
 * FIXME rename TheParam to something better
 * @apiParam {String} TheParam Documentation for the param
 * @apiSuccess {Object} SomeData
 */
```
This way, the FIXME comment is embedded in the @apiGroup tag preceding, no matter how many blank lines I add after it.

**Comment by NicolasCARPi** [comment]
I don't think there is. You must add your FIX before the first `@api`.

**Reply by Kaldor37** [reply]
Ok that's what I thought and applied, thank you very much for the anwer !
